### ISSUE TITLE
`hpm --help` fails with SyntaxError

### ISSUE THREAD

**Issue by alanhussey** [matheuss/hpm#1]
```
~ node -v
v4.1.1
```

```
~ hpm
/usr/local/lib/node_modules/hpm-cli/index.js:18
        let msg = chalk.red('You don\'t have HyperTerm installed! :(\n');
        ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
    at node.js:951:3
```

**Comment by matheuss** [comment]
Thanks @alanhussey for reporting this 😊 I'm at `6.x.x.`, in which `use strict` are not mandatory 😅I'm writing tests ASAP, so we don't experience this kind of issues anymore.
Thanks @alisonmonteiro for the quick fix 👌

**Comment by alisonmonteiro** [comment]
@matheuss Anytime! ;)

**Comment by matheuss** [comment]
Thanks man! Fixed in [v1.0.0-RC1](https://github.com/matheuss/hpm/tree/v1.0.0-RC1) 😊
